### PR TITLE
Reactify bus stop show page

### PR DESCRIPTION
**Pull request by erinnachen**
* Minor clean up on React components

### CODE REVIEW

File: app/javascript/packs/bus-route.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+
+import BusStopLink from './bus-stop-link'
+import Direction from './direction'
+
+const BusRoute = (props) => {
+  const busRoute = props.busRoute
```
Comment (jtrudell) [inline app/javascript/packs/bus-route.jsx:9]: This no semi-colons feels weird

File: app/javascript/packs/bus-stop.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <div>
+      <h1>
+        Hey there, {routeName} bus stop
+      </h1>
+      {
+        Object.keys(predictions).map((predictionKey) => {
+          return (
+            <p key={routeName+"-"+predictionKey}>
+              {capitalize(predictionKey)}: {predictions[predictionKey]}
```
Comment (jtrudell) [inline app/javascript/packs/bus-stop.jsx:17]: Does this pull the underscores out of the keys?

File: app/javascript/packs/utils/capitalize.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const capitalize = (str) => {
```
Comment (jtrudell) [inline app/javascript/packs/utils/capitalize.js:1]: str = str.replace("_", " ");

**Review by jtrudell — CHANGES_REQUESTED** [review]
Just want to pull underscores out of the keys for predictions, otherwise looks good

File: app/javascript/packs/utils/capitalize.js (hunk 3 above)
Comment (jtrudell) [reply app/javascript/packs/utils/capitalize.js:1]: You know you want to write a test for this, too. /duck

File: app/javascript/packs/bus-route.jsx (hunk 1 above)
Comment (erinnachen) [reply app/javascript/packs/bus-route.jsx:9]: yeah. I'm going to get eslint up sometime sooooon.

File: app/javascript/packs/bus-stop.jsx (hunk 2 above)
Comment (erinnachen) [reply app/javascript/packs/bus-stop.jsx:17]: nope.

File: app/javascript/packs/utils/capitalize.js (hunk 3 above)
Comment (erinnachen) [reply app/javascript/packs/utils/capitalize.js:1]: hahahaha. it's capitalize not pretty title.

Comment (erinnachen) [reply app/javascript/packs/utils/capitalize.js:1]: I don't write tests.

Comment (jtrudell) [reply app/javascript/packs/utils/capitalize.js:1]: Do you also not look at the page locally after you add a bunch of react components?
burn

Comment (erinnachen) [reply app/javascript/packs/utils/capitalize.js:1]: ?

Comment (jtrudell) [reply app/javascript/packs/utils/capitalize.js:1]: Sigh

**Comment by jtrudell** [comment]
Merged. But we will pull those underscores!

**Pull request merged**